src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.AltSvc.cs

```diff
@@ -18,11 +18,12 @@ public HttpClientHandler_AltSvc_Test(ITestOutputHelper output) : base(output)
         /// <summary>
         /// HTTP/3 tests by default use prenegotiated HTTP/3. To test Alt-Svc upgrades, that must be disabled.
         /// </summary>
-        protected override HttpClient CreateHttpClient()
+        private HttpClient CreateHttpClient(Version version)
         {
-            HttpClientHandler handler = CreateHttpClientHandler(HttpVersion.Version30);
-
-            return CreateHttpClient(handler);
+            var client = base.CreateHttpClient();
+            client.DefaultVersionPolicy = HttpVersionPolicy.RequestVersionOrHigher;
+            client.DefaultRequestVersion = version ?? UseVersion;
```

Comment:
Why fall back to `UseVersion`? Don't we always want to supply `version` here?

Reply:
This is remnant from previous iteration, it shouldn't be here.